extension/webpack.config.js

```diff
@@ -46,4 +46,21 @@ const config = {
   }
 };
 
-module.exports = config;
+// cf. https://github.com/TypeFox/monaco-languageclient/blob/067681adb1c283bfc9a85db989e79bd06599f3a7/docs/guides/troubleshooting.md#webpack-worker-issues
+/**@type {import('webpack').Configuration}*/
+const monacoWorkerConfig = {
+  entry: {
+    editor: './node_modules/@codingame/monaco-vscode-editor-api/esm/vs/editor/editor.worker.js'
+    // textmate: './node_modules/@codingame/monaco-vscode-textmate-service-override/worker.js'
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, './assets/monaco-workers')
```

Comment:
As far as I understand it, this copy paste the worker from the @codingame package to the monaco/assets folder. So technically we should not add this worker to git and should ignore it?

Reply:
Good point, yes!